tools/clang/tools/dxa/dxa.cpp

```diff
@@ -39,6 +39,10 @@ static cl::opt<bool> ListParts("listparts",
                                cl::desc("List parts in input container"),
                                cl::init(false));
 
+static cl::opt<std::string> AddRS("addrs", cl::desc("Add root signature"),
```

Comment:
I just realized, this functionality already exists in dxc.exe with `-setrootsignature`.  Why are we adding it here (and with different name -addrs)?